test/extended/operators/cluster.go

```diff
@@ -108,8 +108,10 @@ func crashloopingContainerCheck(podFilters ...podFilter) {
 				continue
 			}
 			key := fmt.Sprintf("%s/%s", pod.Namespace, pod.Name)
-			names = append(names, key)
-			podsWithProblems[key] = pod
+			if _, ok := pod.Labels["catalogsource.operators.coreos.com/update"]; !ok {
```

Comment:
doesn't this mean we'll lose the ability to detect crashlooping catalog pods?  i think we need a more surgical solution.

also i'm not clear why we're getting to this point.  If the pod is running, we should hit the default `continue` above, right?   So is it tripping one of the other cases?  which one? why?

Reply:
> doesn't this mean we'll lose the ability to detect crashlooping catalog pods?

We have safety measures in place to protect ourselves from crashlooping update pods, see [this](https://github.com/operator-framework/operator-lifecycle-manager/blob/d980191ecdb848c6538b721906dfc993e86edd6f/pkg/controller/registry/reconciler/grpc.go#L266-L269).
Essentially, if the first catalog pod is running successfully, we don't sacrifice that for a potentially problematic update pod. 

> also i'm not clear why we're getting to this point. If the pod is running, we should hit the default continue above, right?

Apolizies for the confusion, it's hitting the `hasFailingContainer` case, updated my comment in the commit message.

Reply:
1) i think you should add this exemption in `hasFailingContainer` so that we can still catch the other cases if they happen
2) please include a comment in the code explaining why this exemption is needed for these pods so someone else who stumbles upon it doesn't have to dig into git history to understand why it's being done

Reply:
> We have safety measures in place to protect ourselves from crashlooping update pods, see this.

that doesn't help us catch the fact that it's happening (raise a signal via a failing test case in our CI)

Reply:
@bparees done, PTAL, thanks!